cryptoservice/certificate.go

```diff
@@ -28,6 +28,13 @@ func GenerateTestingCertificate(signer crypto.Signer, gun string) (*x509.Certifi
 	return generateCertificate(signer, gun, startTime, startTime.AddDate(10, 0, 0))
 }
 
+// GenerateExpiredTestingCertificate generates an expired X509 Certificate from a template, given a GUN.
+// Good enough for tests where expiration does not really matter; do not use if you care about the policy.
+func GenerateExpiredTestingCertificate(signer crypto.Signer, gun string) (*x509.Certificate, error) {
```

Comment:
Maybe it'd make sense to move this method in to one of the test files instead of here? Looks like it's only used in `trustpinning/certs_test.go`.